### PR TITLE
Add possibility to use annotations on classes

### PR DESCRIPTION
**Pull request by vbauer**

### CODE REVIEW

File: feature-matcher-generator/src/test/java/ru/yandex/qatools/processors/matcher/gen/processing/FillMapWithFieldsProcessOnlyFieldsTest.java [hunk 1]
```diff
@@ -28,7 +29,9 @@
 
     @Parameterized.Parameters(name = "{0}")
     public static Collection<ElementKind> data() {
-        return with(ElementKind.values()).remove(equalTo(ElementKind.FIELD));
+        LambdaList<ElementKind> data = with(ElementKind.values());
+        data.removeAll(asList(ElementKind.FIELD, ElementKind.CLASS));
```
Comment (lanwen) [inline feature-matcher-generator/src/test/java/ru/yandex/qatools/processors/matcher/gen/processing/FillMapWithFieldsProcessOnlyFieldsTest.java]: this class should be renamed to show current state (now it process fields and classes)

Comment (vbauer) [inline feature-matcher-generator/src/test/java/ru/yandex/qatools/processors/matcher/gen/processing/FillMapWithFieldsProcessOnlyFieldsTest.java]: Yes, you're right, it was my bad. I've just updated PR.

**Comment by lanwen** [comment]
thanks! Will release it in a few days

**Comment by vbauer** [comment]
Thank you for merge. I hope I will have time to send few more planned PR..

**Pull request merged**